src/NHibernate/Mapping/ByCode/Impl/IdMapper.cs

```diff
@@ -66,6 +67,38 @@ public void Type(IIdentifierType persistentType)
 			}
 		}
 
+		public void Type(System.Type persistentType, object parameters)
```

Comment:
In a(!) future I would like to have a method which accepts `IDictionary<string, object>` for parameters.